docs/src/examples/components/Dropdown/Types/index.tsx

```diff
@@ -5,20 +5,25 @@ import ExampleSection from 'docs/src/components/ComponentDoc/ExampleSection'
 const Types = () => (
   <ExampleSection title="Types">
     <ComponentExample
-      title="Default"
+      title="Selection"
```

Comment:
Why we rename the Default example? If the default is selection, better leave it as default :)

Reply:
I added Selection as it's the same in Semantic. Default is a dropdown that just pops a list, and does not memorise selection :)